graph/telemetry/istio/util/util.go

```diff
@@ -62,12 +62,20 @@ func HandleResponseCode(protocol, httpResponseCode string, grpcResponseStatusOk
 	return grpcResponseStatus
 }
 
-// IsBadTelemetry tests for known issues in generated telemetry given indicative label values.
+// IsBadSourceTelemetry tests for known issues in generated telemetry given indicative label values.
+// 1) source namespace is provided without workload or app
```

Comment:
Should the comment say "**nor** app"?

Reply:
You are right, the wording here is a bit vague.  I won't update it for the PR but I will update it in passing.  Usually "nor" is used with "neither", so something like, "source namespace is provided but with neither workload nor app"

Reply:
actually, I will update it to see if I can snyk to run again, Not sure what the problem is there...